packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-status.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import {
+  OSDMigrationStatus,
+  BLUESTORE,
+  BLUESTORE_RDR,
+} from '@odf/core/constants';
+import { ODF_DR_DOC_HOME } from '@odf/shared/constants/doc';
+import { RedExclamationCircleIcon } from '@odf/shared/status/icons';
+import { CephClusterKind } from '@odf/shared/types';
+import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
+import { ViewDocumentation } from '@odf/shared/utils/doc-utils';
+import { HealthState } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  HealthBody,
+  HealthItem,
+} from '@openshift-console/dynamic-plugin-sdk-internal';
+import { Divider, Flex, FlexItem } from '@patternfly/react-core';
+import { InProgressIcon } from '@patternfly/react-icons';
+import {
+  getCephStoreType,
+  getOSDMigrationStatus,
+} from '../../../../utils/osd-migration';
+
+const calculateOSDMigration = (
+  cephData: CephClusterKind
+): [number, number, number] => {
+  const migratedDevices = getCephStoreType(cephData)?.[BLUESTORE_RDR] || 0;
+  const totalOsd =
+    (getCephStoreType(cephData)?.[BLUESTORE] || 0) + migratedDevices;
+  const percentageComplete =
+    totalOsd !== 0 ? Math.round((migratedDevices / totalOsd) * 100) : 0;
+
+  return [migratedDevices, totalOsd, percentageComplete];
+};
+
+export const OSDMigration: React.FC<OSDMigrationStatusProps> = ({
```

Comment:
to match it with file name and props' type name...
```suggestion
export const OSDMigrationStatus: React.FC<OSDMigrationStatusProps> = ({
```

Reply:
OSDMigrationStatus name used for "enum OSDMigrationStatus" in data-protection.ts and its used here

Reply:
so ? rename the `enum` or rename ur props' type/file name to match with the FC name...

Reply:
done